dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/NetUtilsTest.java

```diff
@@ -115,4 +115,40 @@ public void testIsValidAddress() {
         Assertions.assertFalse(NetUtils.isValidV4Address(address));
     }
 
+    @Test
+    public void giveIpAddress_thenCheckIsValidIPv4Address_thenCheck_expectNormal() {
+        Assertions.assertFalse(NetUtils.isValidIPv4Address(""));
+
+        String ipAddress = "127.0.0.1";
```

Comment:
> can you add a host case?
> 
> BTW, in k8s deploy, the server host like: ![image](https://user-images.githubusercontent.com/11962619/204452728-84281d3c-340d-4a83-a201-90e95c251f76.png)

sure, but the format of the `host` is too broad, i want to just add a fuzzy check. WDYT.

Reply:
> sure, but the format of the `host` is too broad, i want to just add a fuzzy check. WDYT.

What kind of fuzzy check?

Reply:
> > sure, but the format of the `host` is too broad, i want to just add a fuzzy check. WDYT.
> 
> What kind of fuzzy check?

such as just `string:port`, BTW, may we can rollback these validator, because this validator just for filter `default`, under normal circumstances, there should be no other illegal worker configurations. WDYT. cc @caishunfeng @ruanwenjun @Radeity